pkg/minikube/bootstrapper/bsutil/kverify/kubelet.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package kverify verifies a running Kubernetes cluster is healthy
+package kverify
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/docker/machine/libmachine/state"
+	"github.com/golang/glog"
+	"k8s.io/minikube/pkg/minikube/command"
+	"k8s.io/minikube/pkg/minikube/sysinit"
+	"k8s.io/minikube/pkg/util/retry"
+)
+
+// KubeletStatus checks the kubelet status
+func KubeletStatus(cr command.Runner) state.State {
```

Comment:
If we added `Status` as a function to systemd then it could be used for all services instead of special casing kubelet

Then, we should be able to use something like this instead

https://github.com/kubernetes/minikube/blob/5ce9d99d68a57f833b7af035c384479122479968/pkg/minikube/sysinit/systemd.go#L44